src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/model/OcrDataField.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class OcrDataField {
+
+    @JsonProperty(value = "metadata_field_name")
+    public String name;
+    @JsonProperty(value = "metadata_field_value")
```

Comment:
those are used for serialization, are they needed?

Reply:
no, I removed them